docs/installation_guide.rst

```diff
@@ -151,6 +151,20 @@ At this point the rotki docker container should be running and you
 should be able to access rotki frontend, open your browser and go to
 it at ``http://localhost:8084``. You should be able to see the rotki login screen.
 
+Time in Profit / Loss Report is in wrong.
```

Comment:
This change doesn't affect only to the Profit and Loss Report but to every action. I believe a better title is

```suggestion
Modify the timezone used by rotki
```

Reply:
The UI is running in the correct time zone also with no timezone set in the container.
But your title should also be fine.